tests/test_covariances.py

```diff
@@ -11,11 +11,14 @@ def setUp(self):
 
         self.x = np.linspace(-1, 1, 1000)[:, np.newaxis]
 
-    def _test_compare_sklearn(self, cov: skfda.misc.covariances.Covariance):
+    def _test_compare_sklearn(self, cov: skfda.misc.covariances.Covariance,
+                              eval_y=True):
```

Comment:
Why is this?

Reply:
Sklearn's `WhiteKernel` returns a full-zero array if the second parameter `Y` when calling is not None, so as it is different in the rest of tests it is necessary to pass that flag or do something similar.

Reply:
Is not possible then to pass only `X` in every covariance instead?

Reply:
Changed. Now every covariance is tested passing a single x parameter.